src/components/dropdown.js

```diff
@@ -1,23 +1,39 @@
-import React, { useState } from 'react';
+import React, {useEffect, useState} from 'react';
 import Arrow from '../assets/images/angle_down.svg';
 
-export default function Dropdown({title, options, defaultOption, onClickOption}) {
+export default function Dropdown({options, disable, defaultOption, onClickOption}) {
   
-  const [_title, setTitle] = useState(title)
+  const [_title, setTitle] = useState(defaultOption.name)
   const [displayMenu, setDisplayMenu] = useState(false)
 
   const toggleDisplayMenu = () => {
     setDisplayMenu(!displayMenu)
```

Comment:
I know you didn't write this but for preventing race conditions associated with closures the code should be written as follows:
```
const toggleDisplayMenu = () => setDisplayMenu((prevState) => !prevState)
```